apps/browser/supabase/migrations/20231206012608_chat_history.sql

```diff
@@ -0,0 +1,83 @@
+drop table roles CASCADE;
+drop table chats CASCADE;
+drop table messages CASCADE;
+
+CREATE TABLE "public"."chats" (
+  "id" "uuid" DEFAULT "extensions"."uuid_generate_v4"() NOT NULL,
+  "created_at" timestamp with time zone DEFAULT "now"() NOT NULL,
```

Comment:
NIT: I don't like how we're inconsistent with casing for SQL keywords. For example, we say `NOT NULL` and `not null` in the same file. I think all keywords should be upper case, so for example `DROP TABLE` instead of `drop table`.